pac4j-oauth/src/main/java/org/pac4j/oauth/client/FigShareClient.java

```diff
@@ -0,0 +1,37 @@
+package org.pac4j.oauth.client;
+
+import org.pac4j.core.redirect.RedirectAction;
+
+import org.pac4j.oauth.profile.figshare.FigShareProfile;
+import org.pac4j.oauth.profile.figshare.FigShareProfileCreator;
+import org.pac4j.oauth.profile.figshare.FigShareProfileDefinition;
+import org.pac4j.scribe.builder.api.FigShareApi20;
+
+/**
+ * <p>This class is the OAuth client to authenticate users in FigShare (using OAuth 2.0 protocol).</p>
+ * <p>It returns a {@link com.biovista.org.pac4j.oauth.profile.figshare.FigShareProfile}.</p>
+ * <p>More information at https://docs.figshare.com/old_docs/oauth/</p>
+ *
+ * @author Vassilis Virivilis
+ * @since 3.8
```

Comment:
Sorry fo the change version, but the 3.8.0 has been released. The next version will be the 3.9.0.

Reply:
Forget about the version, I'll release a version 3.8.1.